foundation/foundation-mda/src/main/resources/templates/general-docker/inference.docker.pom.xml.vm

```diff
@@ -90,6 +90,9 @@
             <plugin>
                 <groupId>${group.fabric8.plugin}</groupId>
                 <artifactId>docker-maven-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
```

Comment:
Q: Temporary until the follow on is handled? If so, would be good to add a comment